distribution/sdk/sample-nsili-server/src/main/java/org/codice/alliance/nsili/mockserver/server/MockNsili.java

```diff
@@ -167,17 +167,16 @@ private ORB getOrbForServer(int port)
 
     public static void main(String args[]) {
         if (args.length != 1) {
-            System.out.println("ERROR: Cannot start the mock NSILI server; No ports specified."
-                    + "\nProvide arguments in format: HTTP_PORT,FTP_PORT,CORBA_PORT");
+            LOGGER.error(
+                    "Cannot start the mock NSILI server; No ports specified \nProvide arguments in format: HTTP_PORT,FTP_PORT,CORBA_PORT");
```

Comment:
I think we would want this to stay System.out as it's printed from a `main` to tell the user how to launch the process.